src/evaluator.ts

```diff
@@ -152,17 +152,31 @@ export class Evaluator {
           return true;
         }
 
-        // Should Target be included via segment rules
-        if (
-          segment.rules &&
-          (await this.evaluateClauses(segment.rules, target))
-        ) {
-          this.log.debug(
-            'Target %s included in segment %s via rules\n',
-            target.name,
-            segment.name,
+        if (segment.servingRules && segment.servingRules.length > 0) {
```

Comment:
You can simplify a wee bit
```suggestion
        if (!!segment?.servingRules?.length) {
```

Reply:
I'm going to leave off those `!!` because WebStrom is flagging that as redundant

<img width="588" alt="image" src="https://github.com/harness/ff-nodejs-server-sdk/assets/115623869/3b7e8736-a7fa-47e7-9d64-3b4740d10b22">

Reply:
I'm too used to JSX and needing to make sure things are deffo boolean and not just truthy. Good they can be left out 👍🏻